Bottom: 3152d68a56abd0d301d2166888983b78e2fed2c4
Top:    d9c965f77d68784cc6e8ce7f1f1597f20d435eb7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-03 13:01:51 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
